pkg/cloudprovider/vsphereparavirtual/ippoolmanager/v1alpha2/ippoolmanager.go

```diff
@@ -195,6 +194,6 @@ func (p *IPPoolManager) GetIPPoolListerSynced() cache.InformerSynced {
 }
 
 // StartIPPoolInformers starts ippool informer
-func (p *IPPoolManager) StartIPPoolInformers() {
-	p.informerFactory.Start(wait.NeverStop)
+func (p *IPPoolManager) StartIPPoolInformers(stopCh <-chan struct{}) {
```

Comment:
Do we still need to have this entire file?

IIRC, we are not using IPPool CR anymore when NSX VPC enabled?

Reply:
v1apha2 API was totally removed from nsx-operator just now, I will remove the whole file